src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1590,6 +1590,45 @@ protected override void HandleSetBreakpointsRequestAsync(IRequestResponder<SetBr
                     var resBreakpoints = new List<Breakpoint>();
                     foreach (var bp in breakpoints)
                     {
+                        if (dict.ContainsKey(bp.Line))
```

Comment:
would dict.TryGetValue(key, out value) be a better way to do this in case this becomes async?